app/src/sharedTest/java/org/oppia/app/profileprogress/ProfileProgressFragmentTest.kt

```diff
@@ -109,6 +127,55 @@ class ProfileProgressFragmentTest {
     }
   }
 
+  @Test
+  fun testProfileProgressFragment_openProfilePictureEditDialog() {
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(0)).use {
+      onView(
+        atPositionOnView(
+          R.id.profile_progress_list,
+          0,
+          R.id.profile_edit_image
+        )
+      ).perform(click())
+      onView(withText(R.string.profile_progress_edit_dialog_title)).check(matches(ViewMatchers.isDisplayed()))
+    }
+  }
+
+  @Test
+  fun testAddProfileActivity_imageSelectAvatar_checkGalleryIntent() {
+    val expectedIntent: Matcher<Intent> = allOf(
+      hasAction(Intent.ACTION_PICK),
+      hasData(MediaStore.Images.Media.EXTERNAL_CONTENT_URI)
+    )
+    val activityResult = createGalleryPickActivityResultStub()
+    intending(expectedIntent).respondWith(activityResult)
+    launch<ProfileProgressActivity>(createProfileProgressActivityIntent(0)).use {
+      onView(
+        atPositionOnView(
+          R.id.profile_progress_list,
+          0,
+          R.id.profile_edit_image
+        )
+      ).perform(click())
+      onView(withText(R.string.profile_progress_edit_dialog_title)).check(matches(ViewMatchers.isDisplayed()))
+      onView(withId(R.string.profile_picture_edit_alert_dialog_choose_from_library)).perform(click())
```

Comment:
This test cases is failing because of this line.
Please make sure that all your test cases pass before assigning it for review.